examples/pico-examples/hello.py

```diff
@@ -0,0 +1,35 @@
+# SPDX-FileCopyrightText: 2021 Jeff Epler, written for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+#
+# Adapted from the example https://github.com/raspberrypi/pico-examples/tree/master/pio/hello_pio
+
+import time
+import board
+import rp2pio
+import adafruit_pioasm
+
+hello = """
+.program hello
+loop:
+    pull
+    out pins, 1
+    jmp loop
```

Comment:
Please add a comment about what this does and note that the jmp loop isn't needed. I'm afraid folks will start with this example and not rely on wrap.